src/server/src/main/java/io/cassandrareaper/storage/CassandraStorage.java

```diff
@@ -410,6 +425,41 @@ private void prepareMetricStatements() {
         + " WHERE time_partition = ? AND run_id = ? AND node = ?");
     delNodeMetricsByNodePrepStmt = session.prepare("DELETE FROM node_metrics_v1"
         + " WHERE time_partition = ? AND run_id = ? AND node = ?");
+    storeMetricsPrepStmt
+        = session
+            .prepare(
+                "INSERT INTO node_metrics_v2 (cluster, metric_domain, metric_type, time_bucket, "
+                    + "host, metric_scope, metric_name, ts, metric_attribute, value) "
+                    + "VALUES(?, ?, ?, ?, ?, ?, ?, ?, ?, ?)")
+            .setConsistencyLevel(ConsistencyLevel.LOCAL_ONE);
+    getMetricsForHostPrepStmt
+        = session
+            .prepare(
+                "SELECT cluster, metric_domain, metric_type, time_bucket, host, "
+                    + "metric_scope, metric_name, ts, metric_attribute, value "
+                    + "FROM node_metrics_v2 "
+                    + "WHERE metric_domain = ? and metric_type = ? and cluster = ? and time_bucket = ? and host = ?")
```

Comment:
if the `ts` column were the first clustering column (for the sake of TWCS): then the range operator against `since` could be included here?

Reply:
it could but I'd lose the ability to get a metric for all nodes in the cluster. The `time_bucket` is more important for TWCS than `ts`'s position is here IMHO.
I've changed the TWCS window size to match the hourly bucketing scheme.